### ISSUE TITLE
make test

### ISSUE THREAD

**Issue by atasoglou** [kyamagu/mexopencv#375]
`addpath(pwd);cd test;try,UnitTest('ContribModules',,'Verbosity',2);catch e,disp(e.getReport);end;exit;`

Is this correct: `UnitTest('ContribModules',,'Verbosity',2)`?

**Comment by amroamroamro** [comment]
You're right, it should be replaced by either true of false depending on whether `WITH_CONTRIB` was defined:

```sh
# there are two use cases:
$ make test
$ make WITH_CONTRIB=true test
```

I'll include a fix in the next update. In the mean time, you can always directly call `UnitTest` inside MATLAB with the correct parameters.

**Comment by amroamroamro** [comment]
fixed in latest version (3.3.1)
